src/Samples/SarifToCsv/CsvWriter.cs

```diff
@@ -101,6 +102,11 @@ public void Write(int value)
             Write(value.ToString());
         }
 
+        public void Write(BigInteger value)
+        {
+            Write(value.ToString());
```

Comment:
>value [](http://example.com/codeflow?start=18&length=5)

is there an overload for specifying the invariant culture here>

Reply:
Prefer CultureInfo.InvariantCulture unless default culture is already getting picked up (IOW don't introduce a breaking change in behavior).

Reply:
the soucre code is 
```csharp
public override String ToString() {
            return BigNumber.FormatBigInteger(this, null, NumberFormatInfo.CurrentInfo);
        }
```

Reply:
Got it. So we should actually not use this helper. The reason is that you're persisting to CSV, where we should prefer a stable format.

Imagine someone who produces this CSV on a German system and reading it in the US. Commas and decimals in code may be reversed and cause problems.

Instead of ToString(), use this: BigNumberFormatBigInteger(value, null, NumberFormatInfo.InvariantInfo.

Reply:
I checked we have codes like 
`region.EndLine.ToString(CultureInfo.InvariantCulture)`
I used the same.
This file is actually a sample project code.